File: class_tool/src/main/java/com/tempsfteam/class_tool/dto/LoginDTO.java
package com.tempsfteam.class_tool.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author ADACHI
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class LoginDTO {
    private String name;
    private String password;
    private String uncheckedCode;

    /**
     * 验证码对应的唯一标识
     */
    private String uuid;

}

File: class_tool/src/main/resources/static/src/models/dto/loginDTO.js
export default {
    name: 'loginDTO',
    title: 'LoginDTO',
    fields: [
        { name: 'name', label: '用户名', type: 'string', required: true },
        { name: 'password', label: '密码', type: 'string', required: true, inputType: 'password' },
        { name: 'uncheckedCode', label: '验证码（后端字段）', type: 'string' },
        { name: 'uuid', label: '验证码 UUID', type: 'string' }
    ]
};